src/parachain/oracle.ts

```diff
@@ -93,6 +93,14 @@ export class DefaultOracleAPI implements OracleAPI {
     ) { }
 
     async getExchangeRate(currency: CurrencyExt): Promise<ExchangeRate<Bitcoin, CurrencyExt>> {
+        // KBTC / IBTC have an exchange rate of one
+        if (currency.ticker == this.wrappedCurrency.ticker) {
+            return new ExchangeRate<WrappedCurrency, CurrencyExt>(
+                currency,
+                currency,
+                new Big(1),
+            );
+        }
```

Comment:
What is the advantage of hardcoding this?

Reply:
There is no exchange rate of one for the wrapped currency stored in the aggregate field of the oracle pallet, so it needs to be hardcoded here